### ISSUE TITLE
CMS, components, design system [SCREENREADER] Aside MUST not be contained in another landmark

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#7844]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

On Fiduciary claims — https://preview-prod.vfs.va.gov/preview?nodeId=3016, the entirety of the content between the header and footer is contained in a `<main>` element. The `<main>` element should be reserved for the primary content on the page, and an `<aside>` would be outside of it, at the same code hierarchy level.

This issue appears throughout VA.gov, but is obfuscated because it is usually in a `<div id/class="subnav/sidenav/aside">` — which reveals the purpose of the section, but makes it unavailable to screen reader users.

### Why it Matters

Embedding an `<aside>` element in another landmark may disable screen reader functionality allowing users to navigate through complementary content.

Complementary content is ancillary content to the main theme of a document or page. Screen reader users have the option to skip over complementary content when it appears at the top level of the accessibility API. 

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the page content by landmarks, without missing any content.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: all

## Steps to Recreate

1. Enter https://preview-prod.vfs.va.gov/preview?nodeId=3016 in browser
2. First, run an axe browser scan to see the error appear in the log
3. Second, start the screen reader of your choice
4. Navigate by landmarks
5. Verify that the aside content is not available

## Possible Fixes (optional)

This requires page template and component changes across VA.gov pages. 

Page structure would align with the guidance in the CSS Tricks article  listed in WCAG Guidance. Something along the lines of:

```html
<header>
    <a href="#">Logo</a>
    <nav>Main Nav</nav>
</header>
<section>
    <nav aria-label="breadcrumbs">Breadcrumbs</nav>
    <aside aria-label="secondary" class="va-nav-linkslist va-nav-linkslist--related">Leftnav/sidenav</aside>
    <main>
        The primary content of the page
        <article>Areas of content within the page</article>
    </main>
    <aside>If there is no leftnav, there may be a right rail here, but only one aside on a page</aside>
</section>
<footer>Footer content</footer>
```

## WCAG or Vendor Guidance (optional)

* [CSS Tricks - How to structure your HTML](https://css-tricks.com/how-to-section-your-html/)
* [DequeUniversity - Complementary landmarks and/or asides are top level](https://dequeuniversity.com/rules/axe/3.3/landmark-complementary-is-top-level)

**Comment by laflannery** [comment]
Closing as having a nested aside it valid, it doesn't have to be a top-level landmark. [See more detailed comment here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18694#issuecomment-1803999126)
